server/app/assets/javascripts/trusted_intermediary.ts

```diff
@@ -0,0 +1,25 @@
+// Javascript handling for the trusted intermediary page
+
+import {addEventListenerToElements} from './util'
+
+export function init() {
+  updateListeners()
+}
+
+/** Safe to call multiple times */
+export function updateListeners() {
+  addEventListenerToElements('#cf-ti-clear-search', 'click', resetSearchForm)
+}
+
+/** In the enumerator form - add a new input field for a repeated entity. */
+function resetSearchForm() {
```

Comment:
question (blocking): do tests already exist for this functionality? if not, would you add one that tests clicking that button clears out the inputs?

Reply:
They do, and I know because I broke them on my first try :)

https://github.com/civiform/civiform/blob/1cbcf6ad002700967cd0d1ba0be04e3f106b251d/browser-test/src/trusted_intermediary.test.ts#L570